tests/postgres_tests/test_serialfield.py

```diff
@@ -0,0 +1,64 @@
+from tests.postgres_tests import PostgreSQLTestCase
+from tests.postgres_tests.models import SerialModel
```

Comment:
These should use relative imports (`from . import PostgreSQLTestCase`) and import django components before local test components (see https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports)